scripts/f2grfn_standalone.py

```diff
@@ -74,7 +73,7 @@
         processing_modules,
     ) = f2grfn.fortran_to_grfn(
         args.file,
-        args.directory,
+        args.directory[0],
```

Comment:
@hlim1 If you do the change in my previous comment, this will need to be reverted back to `args.directory,`.